docs/coordinates/inplace.rst

```diff
@@ -22,14 +27,21 @@ you modify the data in the representation the frame contains it is important to
 clear this cache so future calculations of new representations are recomputed
 using the new data. This can be achieved by doing::
 
-      del c.frame.cache['representation']
+    >>> del c.cache['representation']
 
 
 It should be noted that the only way to modify the data in a frame is by using
 the ``.data`` attribute directly and not the aliases for components on the frame
 i.e.::
 
-    c.ra[:] = 20 * u.deg
+    >>> c.ra[()] = 20 * u.deg
 
 will not work as a different representation object is used when acessing the
-aliased component names.
+aliased component names. If you wish to inspect the mapping between frame
```

Comment:
I think that splitting a sentence with a code fragment is difficult to read, and it's usually better to say what's important about a code snippet as a complete sentence prior to the snippet.  So here:  "... for components on the frame.  For instance the following example will not work ... component names::".  

Likewise in the next bit, say " ... you can look at the following dictionary::".